core/block_validator.go

```diff
@@ -247,6 +248,26 @@ func ValidateHeader(config *ChainConfig, pow pow.PoW, header *types.Header, pare
 			return &BlockNonceErr{header.Number, header.Hash(), header.Nonce.Uint64()}
 		}
 	}
+	// DAO hard-fork extension to the header validity: a) if the node is no-fork,
```

Comment:
This should be documented, at least this was agreed upon for about a week now. If it's not, I'd bug people pushing out official specs to include it. Not sure who that is. I can also ask him/her if I find out. Will try.